### ISSUE TITLE
Is it possible to continue generation being artificially terminated?

### ISSUE THREAD

**Issue by zitianlll** [princeton-vl/infinigen#213]
**Describe the feature you would like to see implemented**

If I have to kill the generation process and some of scene have frames(`Image`, rendering metadata such as `DiffCol` and part of gt such as(Depth). Is it possible to continue the generation after interruption?

**Comment by araistrick** [comment]
Yes, you can attempt to rerun the same command with --use_existing and it won't repeat any job which was marked as fully finished (IE any job for which seed/logs/FINISH_$jobtype.txt exists). It may repeat some work for jobs that were midway through.
